test/clj/rems/test/themes.clj

```diff
@@ -1,40 +1,38 @@
 (ns rems.test.themes
   (:require [clojure.test :refer :all]
-            [mount.core :as mount]
-            [rems.test.testing :refer [suppress-logging]]
-            [rems.themes :as themes]
+            [rems.config :as config]
             [rems.util :as util]))
 
-(use-fixtures :once
-  (suppress-logging "rems.themes")
-  (fn [f]
-    (f)
-    (mount/stop)))
+(deftest load-external-theme-test
+  (testing "no theme gives default theme"
+    (let [config {:theme-path nil
+                  :theme {:default "foo"}}]
+      (is (= config (config/load-external-theme config)))))
 
-(deftest load-theme-test
-  (let [default-theme {:default "foo"}]
-    (testing "no theme gives default theme"
-      (is (= default-theme
-             (themes/load-theme nil default-theme))))
-    (testing "non-existing theme gives default theme"
-      (is (= default-theme
-             (themes/load-theme "no-such-file.edn" default-theme))))
-    (testing "static resources can be placed in 'public' directory next to the theme file"
-      (is (= "lbr-theme/public"
-             (:theme-static-resources (themes/load-theme "lbr-theme/lbr.edn" default-theme)))))
-    (testing "overrides values from the default theme"
+  (testing "non-existing theme file is ignored silently"
```

Comment:
why? The user experience is a lot better when there is actually a row in the log explaining why it was ignored.

Reply:
How about just making it an error which prevents the application from starting? That would make detecting the problem even easier.